openfortivpn-webview-electron/index.js

```diff
@@ -51,6 +55,20 @@ const urlBuilder = () => {
 
 const urlRegex = RegExp(argv['url-regex'] ? argv['url-regex'] : defaultUrlRegex);
 
+app.on('certificate-error', (event, webContents, url, error, certificate, callback) => {
+  if (argv['trusted-cert'] === certificate.fingerprint) { 
+    event.preventDefault();
+    callback(true);
+  } else {
+    console.log('Found an invalid certificate:');
+    console.dir(certificate, { depth: null });
+    console.log();
+    console.log('If you know that this certificate can be trusted, relaunch the application passing the following argument to ignore the error:');
+    console.log(`--trusted-cert='${certificate.fingerprint}'`);
```

Comment:
Sorry, one more thing. I'd add this at the top:
```js
const { Console } = require("console");

const errorConsole = new Console(process.stderr);
```

and here I would use:
```js
    errorConsole.log('Found an invalid certificate:');
    errorConsole.dir(certificate, { depth: null });
    errorConsole.log();
    errorConsole.log('If you know that this certificate can be trusted, relaunch the application passing the following argument to ignore the error:');
    errorConsole.log(`--trusted-certs="${certificate.fingerprint}"`);
```

The reason for that is that I want to keep `stdout` for actual outputs, especially because it is likely piped to some other program. Here I could have used `util.inspect()` instead of `Console.dir()`, but then I would've needed to detect if the current terminal supports colors to enable them conditionally, while `Console.dir()` does that automatically.